src/components/sprite-selector/sprite-selector.jsx

```diff
@@ -14,7 +14,7 @@ const addSpriteMessage = (
     <FormattedMessage
         defaultMessage="Add Sprite"
         description="Button to add a sprite in the target pane"
-        id="targetPane.addSprite"
+        id="giu.SpriteSelectorComponent.targetPaneAddSprite"
```

Comment:
is `targetPane` useful? Should this just be `gui.SpriteSelectorComponent.addSprite`

Reply:
note gui typo

Reply:
gui.spriteSelector.addSprite